test/check-multi-machine

```diff
@@ -87,6 +87,11 @@ class TestMultiMachine(MachineCase):
         self.machine3.wait_boot()
         self.machine3.execute("hostnamectl set-hostname machine3")
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1297745
+        if self.machine.image == "rhel-atomic":
```

Comment:
> Yes, there are tradeoffs. But in reality this is broken for real users. We don't want Cockpit to show as green for a given OS, and then have those features fail in real life.

Very true, and I was saying that I didn't want to do workarounds anymore, didn't I?  I'll create the nauhgty patterns.